doc/install-on-macosx.md

```diff
@@ -34,8 +34,8 @@ source setup-env.sh # Add Ray to your Python path.
 ```
 
 For convenience, you may also want to add the line `source
-"$RAY_ROOT/setup-env.sh"` to your `~/.bashrc` file manually, where `$RAY_ROOT`
-is the Ray directory (e.g., `/home/ubuntu/ray`).
+"$RAY_ROOT/setup-env.sh"` to the bottom of your `~/.bashrc` file manually, where
```

Comment:
It doesn't, but typically programs (and people) add this kind of stuff at the bottom. There is already lots of commands in the file, and the ones at the top are typically used by the system.